core/templates/services/state-interaction-stats.service.ts

```diff
@@ -99,7 +93,10 @@ export class StateInteractionStatsService {
    * Returns a promise which will provide details of the given state's
    * answer-statistics.
    */
-  computeStats(state: State): Promise<IStateRulesStats> {
+  computeStats(state: State, cacheResults: boolean): Promise<IStateRulesStats> {
```

Comment:
Perhaps useCachedResults or useCachedResultsIfAvailable -- the semantics of the current name seems like an instruction to cache the results, which does not seem like how this is being used.